packages/react-native-bpk-component-rating/src/NativeRating.android.js

```diff
@@ -20,16 +20,11 @@
 import React from 'react';
 import { requireNativeComponent } from 'react-native';
 
-import { type NativeProps } from './common-types';
+import { defaultProps, type NativeProps } from './common-types';
 
 const AndroidBPKRatingView = requireNativeComponent('AndroidBPKRatingView');
 
-const normalizedIconName = icon => `bpk_${icon.replace(/-/g, '_')}`;
-
-const BpkRating = (props: NativeProps) => {
-  const { icon, ...rest } = props;
-  const normalizedIcon = icon ? icon.map(normalizedIconName) : undefined;
-  return <AndroidBPKRatingView {...rest} icon={normalizedIcon} />;
-};
+const BpkRating = (props: NativeProps) => <AndroidBPKRatingView {...props} />;
+BpkRating.defaultProps = defaultProps;
```

Comment:
This shouldn't have any default props, they come from the `BpkRating` component that wraps this one. Also `NativeProps` doesn't define nullable prop that needs to come from default props.

Reply:
Not including these breaks Flow

Reply:
![image](https://user-images.githubusercontent.com/1333960/68311380-61e39800-00a9-11ea-8b93-e7a697c9054a.png)

Reply:
What is the error? It wasn't breaking before and you only removed props 🤔

Reply:
I fixed it by changing `size?` to `size` and `orientation?` to `orientation`. This is correct because default props are resolved before the function is called so by the time it is `size` and `orientation` will definitely be there

Reply:
you are right 👍